contract/tools/ui-kit-goals/makeHttpClient.js

```diff
@@ -45,6 +45,7 @@ export const makeHttpClient = (url, fetch) => {
     execute: async request => {
       const settings = {
         method: 'POST',
+        mode: 'no-cors',
```

Comment:
Is `no-cors` a reasonable thing to use in all circumstances? What are the security considerations?

Reply:
What does `no-cors` do?